certbot/docs/using.rst

```diff
@@ -529,6 +529,70 @@ revocation from any ACME account::
 
   certbot revoke --cert-path /etc/letsencrypt/live/example.com/cert.pem --key-path /etc/letsencrypt/live/example.com/privkey.pem
 
+.. _deleting:
+
+Deleting certificates
+---------------------
+
+If you need to delete a certificate, use the ``delete`` subcommand.
+
+.. note:: Read this and the `Safely deleting certificates`_ sections carefully. This is an irreversible operation and must
+          be done with care.
+
+.. note:: Do not manually delete certificate files from inside ``/etc/letsencrypt/``. Always use the ``delete`` subcommand.
+
+A certificate may be deleted by providing its name with ``--cert-name``. \
+You may find its name using ``certbot certificates``.
+
+Otherwise, you will be prompted to choose one or more
+certificates to delete::
+
+  certbot delete --cert-name example.com
+  # or to choose from a list:
+  certbot delete
+
+Safely deleting certificates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Before* deleting a certificate, make sure to remove all references to that certificate from the configuration
+of any installed server software (Apache, nginx, Postfix, etc). Otherwise, you may end up with a non-functioning
+server.
+
+When installing a certificate, Certbot modifies Apache or nginx's configuration to load the certificate and its
+private key from the ``/etc/letsencrypt/live/`` directory.
+
+Before deleting a certificate, it is necessary to undo that modification, by removing any references to the certificate
+from the webserver's configuration files:
+
+1. Find all references to the certificate (substitute ``example.com`` in the command for the name of the certificate
+   you wish to delete)::
+
+     sudo bash -c 'grep -R live/example.com /etc/{nginx,httpd,apache2}'
+
+   If there are no references found, skip directly to Step 4.
+
+   If some references are found, they will look something like::
+
+     /etc/apache2/sites-available/000-default-le-ssl.conf:SSLCertificateFile /etc/letsencrypt/live/example.com/fullchain.pem
+     /etc/apache2/sites-available/000-default-le-ssl.conf:SSLCertificateKeyFile /etc/letsencrypt/live/example.com/privkey.pem
+
+2. Generate a self-signed certificate to use as a substitute::
+
+     sudo openssl req -nodes -batch -x509 -newkey rsa:2048 -keyout /etc/letsencrypt/self-signed-privkey.pem -out /etc/letsencrypt/self-signed-cert.pem -days 356
+
```

Comment:
```suggestion

This command will place a self-signed certificate at `/etc/letsencrypt/self-signed-cert.pem`, and the key used to sign that certificate at `'etc/letsencrypt/self-signed-privkey.pem`.

```

I know we imply that in the next step, but I think it could use additional clarity in the place where it happens, to help build the mental model especially for those who want to modify the steps.